### ISSUE TITLE
Possible to build only 1 of the 11?

### ISSUE THREAD

**Issue by graysky2** [notspiff/screensavers.rsxs#8]
Is it possible to build or to install only 1 of the 11 screensavers via a cmake or a make switch?

Currently building like this:

```
cmake -DCMAKE_INSTALL_PREFIX=/usr -DCMAKE_BUILD_TYPE=Release -DBUILD_SHARED_LIBS=1 -DUSE_LTO=1
make
make DESTDIR="$pkgdir/" install
```

**Comment by notspiff** [comment]
Not via switch. Comment out the build_addon () lines in cmakelists.txt is
the easiest
27. jun. 2016 07:53 skrev "graysky" <notifications@github.com>:

**Comment by graysky2** [comment]
Thanks for the suggestion and for the code!

**Comment by notspiff** [comment]
heh. i forgot to metion that i fully recognize the validity of the request
and have a patch ready but i ran into some other issues i haven't fix yet
(gcc5 compatiblity) so i haven't been able to test it.

**Comment by notspiff** [comment]
i published it in case you want to try before i get around to merging it.
https://github.com/notspiff/screensavers.rsxs/tree/add_subaddon_build_options

this adds -DBUILD_XXX=0/1 for each addon (XXX is upper-cased named of the
sub-addons, -DBUILD_FLOCKS=0 for instance).

**Comment by graysky2** [comment]
My distro is on gcc6.1 and patches your stuff with [this](https://git.archlinux.org/svntogit/community.git/tree/trunk/0001-fix-gcc5-compile.patch?h=packages/kodi-addon-screensaver-rsxs) patch.  Perhaps you can fork over to an unstable branch and commit your patch?  I am happy to test it.

**Comment by notspiff** [comment]
sorry for the long delay. i have added the gcc5 fix through #12 and the ability to build only certain add-ons (a ENABLE_XXX option for each sub-addon). it is tagged as v1.4.0
